jujugui/static/gui/src/app/store/env/controller-api.js

```diff
@@ -1121,7 +1137,12 @@ YUI.add('juju-controller-api', function(Y) {
           const names = credentialTags.map(credentialTag => {
             return tags.parse(tags.CREDENTIAL, credentialTag);
           });
-          return {names: names};
+          const displayNames = names.map(
```

Comment:
Please update the docstring of this method to reflect the presence of a new field.